src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,19 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">
+        {comment.name}
+      </strong>
+      {' by '}
```

Comment:
Wrap it with span

Reply:
@OleziO Word "by" or comment name and "by" ? And why?